### ISSUE TITLE
Focus indicator in input fields can look weird in debug land

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#2509]
@isidorn maybe do a similar solution as outlined in https://github.com/Microsoft/vscode/pull/2497

<img width="518" alt="screen shot 2016-01-28 at 18 15 11" src="https://cloud.githubusercontent.com/assets/900690/12652446/29a9b00c-c5eb-11e5-95be-26e94856766b.png">
<img width="340" alt="screen shot 2016-01-28 at 18 14 58" src="https://cloud.githubusercontent.com/assets/900690/12652447/29ace5ba-c5eb-11e5-913f-5836b05a00fa.png">

**Comment by isidorn** [comment]
@bpasero so not showing focus in this case would be fine? Since that is what the fix you mentioned would do. I guess a blinking cursor is enough of an indication.
Then I would do this for the repl input as well and not show focus since there is the cursor

**Comment by bpasero** [comment]
@isidorn no I suggest to draw the blue around the entire box including the ">". Check the search input field and how it shows the toggles within.

**Comment by bpasero** [comment]
(we decided to not hide the outline even for input fields, so we should not hide it in the repl either)

**Comment by bpasero** [comment]
For the conditional breakpoints we gotta find something that does not look so bad :)

**Comment by isidorn** [comment]
Ok, I will do it for the repl.
Forwarding this one to @bgashler1 to think about how to show conditional breakpoint widget being focussed

**Comment by bpasero** [comment]
@isidorn for the conditional breakpoint try to remove the 1px border from the input box, it looks weird anyways to not align the input field nicely with the sides:

![image](https://cloud.githubusercontent.com/assets/900690/12720270/340b3d7e-c8fa-11e5-97c6-0ecf72ad93d4.png)

When I do that, the blue outline does not show up, but there I am not worried so much because of the large orange indicator.

**Comment by isidorn** [comment]
Sounds good to me, orange should be a clear focus indicator

**Comment by isidorn** [comment]
Ok I am no longer showing the blue focus as we agreed. The issue now is that if the user open the conditional breakpoint widget and clicks somewhere else the outline is equally orange, so the user might not feel like he lost focus.

I can bring this up in our weekly UX sync where we should discuss accessibility and focus anyway

fyi @stevencl @bgashler1

**Comment by bpasero** [comment]
@isidorn yup, would leave this one open, also because of REPL.

**Comment by isidorn** [comment]
Repl is a separate bug which I just now fixed #2536 
Let me know if I should still reopen this :)

**Comment by bpasero** [comment]
:+1:
